scss/_variables.scss

```diff
@@ -286,6 +286,18 @@ $grid-hovered-bg: $table-bg-hover !default;
 $grid-hovered-border: $grid-border !default;
 
 
+// Progress
+$progress-text: $body-color !default;
+$progress-bg: $progress-bg !default;
+$progress-border: transparent !default;
+$progress-gradient: none !default;
+
+$progress-selected-text: $progress-bar-color !default;
+$progress-selected-bg:  $progress-bar-bg !default;
+$progress-selected-border: transparent !default;
+$progress-selected-gradient: none !default;
```

Comment:
I was being consistent in the naming vars. It's not unexpected to have gradient fill for progress bar, be it striped or real gradient.

I agree on the fill part